src/styles/default.css

```diff
@@ -101,23 +101,23 @@ footer {
 }
 
 .header-item {
-  @apply px-2 py-2 text-zinc-hot-pink bg-zinc-dark-purple border-t-2 border-l-2 border-b-2 border-zinc-hot-pink;
+  @apply px-2 py-2 text-zinc-pastel-pink bg-zinc-dark-purple border-t-2 border-l-2 border-r-2 border-b-2 border-zinc-pastel-pink drop-shadow-xl;
```

Comment:
Your addition of `border-r-2` here (and in the `:last-child` variant for `header-item`) is causing double borders to appear between the menu items (the border-r from one item, plus the border-l of the next item).

Reply:
fixed 980772d78ab69824d9a31a07f486501a78044bec TY